test/unit/spec/PasswordExpired_spec.js

```diff
@@ -140,6 +143,67 @@ function (Okta, OktaAuth, LoginUtil, Util, PasswordExpiredForm, Beacon, Expect,
             ' a lowercase letter, an uppercase letter, a number, a symbol, no parts of your username.');
         });
       });
+
+      Expect.describe('Password description in HTML', function () {
+        itp('does not have a subtitle if password requirements as HTML FF is on', function () {
+          return setup({ 'features.showPasswordRequirementsAsHtmlList': true }).then(function (test) {
+            expect(test.form.subtitle().length).toEqual(0);
+          });
+        });
+        itp('shows password requirements as HTML list if FF is on', function () {
+          return setup({ 'features.showPasswordRequirementsAsHtmlList': true }).then(function (test) {
+            expect(test.form.passwordRequirementsHtmlHeader().trimmedText()).toEqual('Password requirements:');
+            expect(test.form.passwordRequirementsHtmlListItems().length).toEqual(8);
+            expect(test.form.passwordRequirementsHtmlListItems().eq(0).text()).toEqual('At least 8 characters');
```

Comment:
nit:
something like below (pseudo code) would simplify code a little bit. 
```
expect(test.form.passwordRequirementsHtmlListItems().map(x => x.text())).toEqual([
 ...
])
```

Reply:
That would work, but I did feel that this was more explicit on the order and in general easy to copy around. I can change it if you have a strong preference.

Reply:
`[2, 3] toEqual [3, 2]` shall be false because it already checks order.
I dont have strong preference though.

Reply:
Spoke offline. Leaving as is.